vin/app/models/package.rb

```diff
@@ -1,30 +0,0 @@
-#enconding: utf-8
-
-class Package < ActiveRecord::Base
-  # Attributes
-  # attr_accessible :type, :note
-
-  # Associations
-  has_many :bottles
-  has_many :wine, through: :bottles
-  belongs_to :shipments
-
-  # Validations
-  validates :mix, inclusion: { in: %w(AR AW RW) }
-
-
-  # Class methods
-  def self.uniq_wines
-    @relation.map { |p| p.uniq_wines }
-  end
-
-  # Methods
-  def uniq_wines
-    self.wine.uniq
-  end
```

Comment:
I'm accessing the wines through bottles, that means i will get, i.e., a reference to the same wine but 6 times! or 3 of one and 3 of other, and so... I was wondering if this should better be a scope, but i never did scopes before, so decided to go for methods.
The relation between the class method and the regular one, is that the class method is meant to work over a collection of `Package`'s, that's why i'm using `map` there